### PR TITLE
Airbnb lint

### PR DESCRIPTION
**Pull request by koosoku**
apply airbnb-base eslint rules and fix the build to pass eslint

### CODE REVIEW

File: src/controllers/users_controller.js [hunk 1]
```diff
@@ -1,108 +1,120 @@
...
-        return res.status(400).json({
-            error: "Email, name and Password must be provided"
-        });
-    }
+module.exports.register = ({ body: { name = null, email = null, password = null } }, res) => {
+  if (!email || !password || !name) {
+    res.status(400).json({
+      error: 'Email, name and Password must be provided',
+    });
+  } else {
```
Comment (singhprincejeet) [inline src/controllers/users_controller.js]: Why are we changing to this approach, or is this just a lint thing

Comment (koosoku) [reply src/controllers/users_controller.js]: this is the standard from the airbnb-base

File: src/controllers/recipes_controller.js [hunk 2]
```diff
@@ -1,97 +1,100 @@
...
-        return res.status(400).json("Please enter the recipe Id");
-    }
 
-    const pythonProcess = spawn("python", ["recommender.py", PYTHON_MODES.RECOMMEND, recipeId]);
+module.exports.recommendRecipe = ({ params: { recipeId = null } }, res) => {
+  if (recipeId === null) {
+    return res.status(400).json('Please enter the recipe Id');
+  }
+
+  const pythonProcess = spawn('python', ['recommender.py',
```
Comment (silverAndroid) [inline src/controllers/recipes_controller.js]: if we're defining an array on separate lines, could the first element also be on a separate line?

File: src/models/users_model.js [hunk 3]
```diff
@@ -1,178 +1,204 @@
...
+  ], callback);
 };
 
 module.exports.findAllUsers = (callback) => {
-    async.waterfall([
-        connect,
-        selectAllUsers
-    ], callback);
+  async.waterfall([
+    connect, selectAllUsers,
```
Comment (silverAndroid) [inline src/models/users_model.js]: for the arrays in this file, could we either have it like `[ ...., ...., ]` or
```
[
    ....,
    ....,
]
```

File: src/services/auth.js [hunk 4]
```diff
@@ -1,40 +1,41 @@
-const bcrypt = require("bcrypt");
+const bcrypt = require('bcrypt');
+
 const saltRounds = 10;
-const jwt = require("jsonwebtoken");
-const {jwtSecret} = require("./config");
+const jwt = require('jsonwebtoken');
+const { jwtSecret } = require('./config');
```
Comment (silverAndroid) [inline src/services/auth.js]: could we have the requires together and `saltRounds` separate?

File: src/models/users_model.js (hunk 3 above)
Comment (koosoku) [reply src/models/users_model.js]: Yeah for sure, I'm surprised that lint didn't complain, but I'll make the change :)

File: src/controllers/recipes_controller.js (hunk 2 above)
Comment (koosoku) [reply src/controllers/recipes_controller.js]: yup, will change!

File: src/services/auth.js (hunk 4 above)
Comment (koosoku) [reply src/services/auth.js]: yupp

**Pull request merged**